js/coinone.js

```diff
@@ -0,0 +1,298 @@
+                    'symbol': 'IOT/KRW',
+                    'base': 'IOT',
+                    'quote': 'KRW',
+                },
+                'LTC/KRW': {
+                    'id': 'ltc',
+                    'symbol': 'LTC/KRW',
+                    'base': 'LTC',
+                    'quote': 'KRW',
+                },
+                'QTUM/KRW': {
+                    'id': 'qtum',
+                    'symbol': 'QTUM/KRW',
+                    'base': 'QTUM',
+                    'quote': 'KRW',
+                },
+                'XRP/KRW': {
+                    'id': 'xrp',
+                    'symbol': 'XRP/KRW',
+                    'base': 'XRP',
+                    'quote': 'KRW',
+                },
+                'ETH/KRW': {
+                    'id': 'eth',
+                    'symbol': 'ETH/KRW',
+                    'base': 'ETH',
+                    'quote': 'KRW',
+                },
+                'ETC/KRW': {
+                    'id': 'etc',
+                    'symbol': 'ETC/KRW',
+                    'base': 'ETC',
+                    'quote': 'KRW',
+                },
+                'BCH/KRW': {
+                    'id': 'bch',
+                    'symbol': 'BCH/KRW',
+                    'base': 'BCH',
+                    'quote': 'KRW',
+                },
+            },
+            'fees': {
+                'trading': {
+                    'tierBased': true,
+                    'percentage': true,
+                    'taker': 0.1 / 100,
+                    'maker': 0.1 / 100,
+                    'tiers': {
+                        'taker': [
+                            [100000000, 0.1 / 100],
+                            [10000000000, 0.09 / 100],
+                            [50000000000, 0.08 / 100],
+                            [100000000000, 0.07 / 100],
+                            [200000000000, 0.06 / 100],
+                            [300000000000, 0.05 / 100],
+                            [400000000000, 0.04 / 100],
+                            [500000000000, 0.03 / 100],
+                            [999900000000000, 0.02 / 100],
+                        ],
+                        'maker': [
+                            [100000000, 0.1 / 100],
+                            [10000000000, 0.08 / 100],
+                            [50000000000, 0.06 / 100],
+                            [100000000000, 0.04 / 100],
+                            [200000000000, 0.02 / 100],
+                            [300000000000, 0.01 / 100],
+                            [400000000000, 0],
+                            [500000000000, 0],
+                            [999900000000000, 0],
+                        ],
+                    },
+                },
+            },
+        });
+    }
+
+    async fetchBalance (params = {}) {
+        let res = await this.privateGetV2AccountBalance ();
+        let result = { 'info': res };
+        Object.values (this.markets).forEach ((mrk) => {
```

Comment:
We can't use forEach here, because it is again not very portable syntactically. This line currently violates the following rule:

> unfold all maps and comprehensions to basic for-loops

This means that we have to rewrite it in terms of 

```JavaScript
for (let i = 0; i < abc.length; i++) {
    // ...
}
```

This is again a portability issue, though this line is ok for JS, we can't use it in language-agnostic code, unfortunately.